ts/api/pagopa.ts

```diff
@@ -134,12 +134,71 @@ type GetWalletsUsingGETExtraT = MapResponseType<
   WalletListResponse
 >;
 
+/**
+ * it sanitizes psp tags avoiding no string value and string duplicates
+ * @param w wallet object
+ */
+const fixWalletPspTagsValues = (w: any) => {
+  const decoder = t.readonly(t.union([t.string, t.unknown]));
```

Comment:
you can use `t.partial` to retain unknown fields:

```suggestion
  const decoder = t.partial({ 
    psp: t.partial({
      tags: t.readonlyArray(t.unknown)
    })
  });
  const decoded = decoder.decode(w);
  // ...
  return {
    ...decoded.value,
    psp: {
      ... decoded.value.psp,
      tags: decoded.value.psp.tags.filter(...)
    }
  }
```

Reply:
@cloudify Since this function has to sanitize tags field (_**wallet.psp.tags**_), can I use this decoder?
```javascript
const decoder = t.interface({
    psp: t.interface({
      tags: t.readonlyArray(t.unknown)
    })
  });
```
In this way if psp and tags fields are not present the decoder will fail.
What do you think?

Reply:
yes makes sense, just one suggestion for readability:

instead of:

```ts
if (decoded.isRight()) {
// ...long block...
return { ... };
}
return w;
```

do

```ts
if(decoded.isLeft()) {
  return w;
}
// code
return { ... } 
```